source/swepub/types.ttl

```diff
@@ -0,0 +1,49 @@
+prefix : <https://id.kb.se/vocab/>
+prefix swepub: <https://id.kb.se/term/swepub/>
+prefix sppubl: <https://id.kb.se/term/swepub/publication/>
+prefix spout: <https://id.kb.se/term/swepub/output/>
+
+swepub:ArtisticWork a :GenreForm ;
+    :exactMatch sppubl:kfu, spout:artistic-work .
+
+swepub:Book a :GenreForm ;
+    :exactMatch sppubl:bok, spout:publication\/book .
+
+swepub:BookChapter a :GenreForm ;
+    :exactMatch sppubl:kap, spout:publication\/book-chapter .
+
+swepub:ConferencePaper a :GenreForm ;
+    :closeMatch sppubl:kon\/ref, spout:conference\/paper .
+
+swepub:DoctoralThesis a :GenreForm ;
+    :exactMatch sppubl:dok, spout:publication\/doctoral-thesis .
+
+swepub:EditorialCollection a :GenreForm ;
+    :code "sam" ;
+    :exactMatch sppubl:sam, spout:publication\/edited-book .
+
+swepub:EditorialProceedings a :GenreForm ;
+    :code "pro" ;
```

Comment:
How come is code only defined for swepub:EditorialCollection and swepub:EditorialProceedings?

Reply:
Good call Actually, code should be removed here. The exactMatch links to the term with the code.